server/lib/AirspaceChecker.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+
+const querystring = require('querystring');
+const fetch = require('isomorphic-fetch');
+
+class AirspaceChecker {
+  /**
+   * Determines if the drone is allowed to fly on this flight path
+   * 
+   * @param {*} origin Array of 2 values: latitude & longitude
+   * @param {*} destination Array of 2 values: latitude & longitude
+   * @returns {boolean} True if the drone can fly on this segment, false if not
+   */
+  static async checkSpace(origin, dest) {
+    let isOK;
+
+    try {
+      const params = {
+        geometry: `{ "type": "LineString", "coordinates": [[${origin[1]}, ${origin[0]}], [${dest[1]}, ${dest[0]}]] }`,
+        types: 'airport',
+        geometry_format: 'geojson',
+        full: true,
+        buffer: 3,
+      };
+      const query = querystring.stringify(params);
+      const options = {
+        headers: {
+          'x-api-key': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjcmVkZW50aWFsX2lkIjoiY3JlZGVudGlhbHwyREp4bE5wQzVleHZRZVVNQkIwTHhTbHBNTW1lIiwiYXBwbGljYXRpb25faWQiOiJhcHBsaWNhdGlvbnw5dzhaS0F2VVc5WUt2eFNCV1lsbEV0dkthbDI1Iiwib3JnYW5pemF0aW9uX2lkIjoiZGV2ZWxvcGVyfFdZV21scGFVbTRvNnBsaDU4RWV2QUhHUkI1WnkiLCJpYXQiOjE0OTk4MjQwNDV9.NJtsnXi4DGDKZDO1DNSmcKvlrVAB50f0ESDXSRZdVa4',
```

Comment:
Best to not have this here, as we discussed.

Reply:
@yanarchy If it's not there currently it breaks the tests, my plan was to move it when deploying to Heroku.

Reply:
Please remove so there isn't a commit on GH with the api-key in there. Move it to a .env file and change the tests.

Reply:
Ok, this should be working now.